reference/constraints.rst

```diff
@@ -52,6 +52,7 @@ Validation Constraints Reference
    constraints/Iban
    constraints/Isbn
    constraints/Issn
+   constraints/Bic
```

Comment:
in case this is not an alphanumeric order, i would put this directly under Iban, because they belong togehter